package.json

```diff
@@ -93,9 +93,11 @@
     "multiselect": "~0.9.12",
     "number_helpers": "^0.1.1",
     "patternfly": "^3.58.0",
-    "patternfly-react": "^2.30.6",
+    "patternfly-react": "^2.32.4",
```

Comment:
https://github.com/theforeman/foreman-packaging/pull/3752

Reply:
changes requested there, waiting on packaging ack here, I retested and functionality wise it's ready :+1: